README.md

```diff
@@ -27,13 +27,22 @@ Read more about the [**basic concepts** of Cucumber that we will be using in thi
 
 All other branches are considered legacy (not under development anymore): `Manager 2.1`, `manager21-longterm`, `Manager 17`, `Manager 12`.
 
+
 # Running the testsuite
 
 You can run the Spacewalk Testsuite [with sumaform](https://github.com/moio/sumaform/blob/master/README_ADVANCED.md#cucumber-testsuite).
 
-## Tests order
+## Core-Features, Idempotency and tests-order.
+
+The test (features) are grouped by: core-features and secondary(idempotent) feature.
```

Comment:
The test"s" (features) are grouped by: core-features and secondary(idempotent) feature"s".

Two "s" missing 😄